task16/index.js

```diff
@@ -261,18 +83,74 @@ const renderTabContainer = (value) => {
 const createArticleTitle = ({ article }) => {
   const articleTitleFragment = document.createDocumentFragment();
   for (let i = 0; i < article.length; i++) {
-    const articleTitleItem = createElementWithClassName("li", "article-title-item");
-    const articleWrapper = createElementWithClassName("article", "article-wrapper");
+    const articleTitleItem = createElementWithClassName(
+      "li",
+      "article-title-item"
+    );
+    const articleWrapper = createElementWithClassName(
+      "article",
+      "article-wrapper"
+    );
     const articleLink = createElementWithClassName("a", "article-link");
     const articleTitle = createElementWithClassName("h1", "article-title");
     articleTitle.textContent = article[i].title;
     articleLink.href = "#";
 
-    articleTitleFragment.appendChild(articleTitleItem).appendChild(articleWrapper).appendChild(articleLink).appendChild(articleTitle);
+    articleTitleFragment
+      .appendChild(articleTitleItem)
+      .appendChild(articleWrapper)
+      .appendChild(articleLink)
+      .appendChild(articleTitle);
+
+    renderNewArrivals(article[i], articleLink);
+    renderCommentCount(article[i], articleLink);
   }
   return articleTitleFragment;
 };
 
+const renderNewArrivals = (article, articleLink) => {
+  const articleDate = elapsedTime(article.date);
```

Comment:
I think it is a difference rather than the date of the article.
ex)Difference days, diffDays ...

Reply:
Thank you for pointing that out. It was weird here too. I was able to fix it.
6c7fbc4